www/_template/reference/configuration.md

```diff
@@ -3,112 +3,284 @@ layout: layouts/content.njk
 title: snowpack.config.js
 description: The Snowpack configuration API reference.
 ---
-
 ```js
-// Example Configuration File
+// Example: snowpack.config.js
 module.exports = {
   plugins: [
     /* ... */
   ],
-  installOptions: {
-    /* ... */
-  },
-  devOptions: {
-    /* ... */
-  },
-  buildOptions: {
-    /* ... */
-  },
-  mount: {
-    /* ... */
-  },
-  alias: {
+};
+```
+
+```js
+// Example: snowpack.config.js (ESM)
+// This is ESM-format config file. to enable
+// Add "type": "module" in your package.json
+export default {
+  plugins: [
     /* ... */
-  },
+  ],
 };
 ```
 
-<!--
-### config.plugins
+> To generate a basic configuration file scaffold in your Snowpack project run `snowpack init`.
 
-- **`extends`** | `string`
-  - Inherit from a separate "base" config. Can be a relative file path, an npm package, or a file within an npm package. Your configuration will be merged on top of the extended base config.
-- **`exclude`** | `string[]`
-  - Exclude any files from scanning, building, etc. Defaults to exclude common test file locations: `['**/node_modules/**/*', '**/__tests__/*', '**/*.@(spec|test).@(js|mjs)']`
-  - Useful for excluding tests and other unnecessary files from the final build. Supports glob pattern matching.
-- **`install`** | `string[]`
-  - Known dependencies to install with Snowpack. Useful for installing packages manually and any dependencies that couldn't be detected by our automatic import scanner (ex: package CSS files).
-- **`mount.*`**
-  - Mount local directories to custom URLs in your built application.
-- **`alias.*`**
-  - Configure import aliases for directories and packages. See the section below for all options.
-- **`proxy.*`**
-  - Configure the dev server to proxy requests. See the section below for all options.
-- **`plugins`**
-  - Extend Snowpack with third-party tools and plugins. See the section below for more info.
-- **`installOptions.*`**
-  - Configure how npm packages are installed. See the section below for all options.
-- **`devOptions.*`**
-  - Configure your dev server. See the section below for all options.
-- **`buildOptions.*`**
-  - Configure your build. See the section below for all options.
-- **`testOptions.*`**
-  - Configure your tests. See the section below for all options. -->
+## config.root
+`string`
+Default: `/`
 
-### config.installOptions
+Specify the root of a project using Snowpack.
 
-Settings that determine how Snowpack handles installing modules.
+Previously config.cwd
 
-#### installOptions.dest `string`
+## config.install
+`string[]`
 
-- _Default:`"web_modules"`_
-- Configure the install directory.
+Known dependencies to install with Snowpack. 
 
-#### installOptions.sourceMap `boolean`
+> Used for installing packages any dependencies that couldn't be detected by our automatic import scanner (ex: package CSS files).
```

Comment:
no blockquotes on the reference page, especially when the main description is only one line. This (and others) can just be normal paragraphs.

Reply:
this is still an issue. Blockquote gradients create too much visual noise in the final page. Here and throughoutk, this is Fine to just be a normal paragraph.